web/components/njwds-extended/VerticalStepIndicator.tsx

```diff
@@ -3,10 +3,12 @@ import { useMountEffect, useOnWindowResize } from "@/lib/utils/helpers";
 
 interface Props {
   number: number;
+  order?: number;
   last: boolean;
   active?: boolean;
   small?: boolean;
   completed?: boolean;
+  openSteps?: number[];
```

Comment:
The thing about `openSteps` is that, now that we have both `order` and `number` props representing different ways of numbering steps, it's not obvious from looking here whether this is referencing the `number` or `order` as the list of open steps. I have to go look to see how it's used in the code to find out. Maybe a rename to something like `openStepNumbers` might be more descriptive to help here